tests/compas/datastructures/test_halfedge.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+
+from compas.datastructures.mesh.core import HalfEdge
```

Comment:
this should import from mesh, not core...

Reply:
actually, from datastructures

Reply:
Updated.